packages/config/config/devices/0x0438/4512757.json

```diff
@@ -0,0 +1,534 @@
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "23",
+			"label": "Temp Unit",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"unit": "°C",
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "°C",
+					"value": 0
+				},
+				{
+					"label": "°F",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "27",
+			"label": "Temp correction",
+			"description": "Allowable range: -10~10",
+			"valueSize": 4,
+			"minValue": -10,
+			"maxValue": 10,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "31",
+			"label": "Work days set",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "5_2",
+					"value": 0
+				},
+				{
+					"label": "6_1",
+					"value": 1
+				},
+				{
+					"label": "7_0",
+					"value": 2
+				},
+				{
+					"label": "0_7",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "40",
+			"label": "Child lock",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"defaultValue": 0,
+			"minValue": 0,
+			"maxValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "43",
+			"label": "Sensor mode",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "A",
+					"value": 0
+				},
+				{
+					"label": "F",
+					"value": 1
+				},
+				{
+					"label": "AF",
+					"value": 2
+				},
+				{
+					"label": "A2",
+					"value": 3
+				},
+				{
+					"label": "A2F",
+					"value": 4
+				},
+				{
+					"label": "P",
+					"value": 6
+				}
+			]
+		},
+		{
+			"#": "58",
+			"label": "Run mode",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"defaultValue": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Manual",
+					"value": 0
+				},
+				{
+					"label": "Program",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "112",
+			"label": "Adaption",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"defaultValue": 0,
+			"minValue": 0,
+			"maxValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "118",
+			"label": "Regulator",
+			"description": "Allowable range: 0-30",
+			"valueSize": 4,
+			"defaultValue": 30,
+			"minValue": 0,
+			"maxValue": 30,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "30M",
+					"value": 30
+				}
+			]
+		},
+		{
+			"#": "123",
+			"label": "LCD Backlight standby",
+			"description": "Allowable range: 10-100 (increment by 10)",
```

Comment:
I'm not sure about these kinds of parameters with increments. We currently have no good way to specify increments aside from the description.
I think what you did is fine.